xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuMacro.xml

```diff
@@ -851,12 +858,6 @@ require(['jquery','xwiki-l10n!menu-ui-translation-keys'], function($, l10n) {
               padding: 0;
               display: inherit;
             }
-            /* Place the arrow on the right */
```

Comment:
This block of style is not used anymore, it can be dropped without consequences.